src/content/api/http-status-codes.mdoc

```diff
@@ -98,7 +98,7 @@ the URL or HTTP method. Please check against examples in our documentation.
 
 ## 429 Too Many Requests
 
-Centrapay API rate limits have been exceeded.
+Centrapay API rate limits have been exceeded. See [Rate Limits](/api/rate-limits) for more details.
```

Comment:
Suggestion: Perhaps we should remove the whole "debugging" section and replace with a link to the new page i.e. move the new sentence here in place of the debugging section.